sematic/ui/packages/tests/fixtures/run.ts

```diff
@@ -26,7 +28,7 @@ const run = {
         "final"
     ],
     "updated_at": new Date("2023-02-03T16:51:47.578636+00:00"),
-    "user": null as any,
+    "user": null as User | null,
```

Comment:
This is what motivates this PR. Fixtures are defined in the `test` package, but it could not reference the type `User` defined in `main` package. Now after the refactoring, it can.